docs/runbooks/VirtualMachineCRCErrors.md

```diff
@@ -14,26 +14,28 @@ major service outages.
 
 ## Diagnosis
 
-1) Get the volume name from the VM:
+Execute the following PromQL expression to get a list of Virtual Machines that
```

Comment:
Suggestion: "Obtain a list of VirtualMachines with an incorrectly configured storage class by running the following PromQL query:"